dungeon/assetlist.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "dungeon",
+  "assets": [
+    {
+      "description": "Dungeon native token",
+      "denom_units": [
+        {
+          "denom": "DGN",
+          "exponent": 6
+        }
+      ],
+      "base": "udgn",
```

Comment:
The base unit must be defined in the denom_unit with exponent 0, but there is no such unit defined, so this will fail checks.